### PR TITLE
Prove hashunsngx, hashunsnggt, and hashgt23el

### PR DESCRIPTION
**Pull request by BTernaryTau**
hashunsngx:
> The size of the union of a set with a disjoint singleton is the extended real addition of the size of the set and 1, analogous to hashunsng.

hashunsnggt:
> The size of a set is greater than a nonnegative integer N if and only if the size of the union of that set with a disjoint singleton is greater than N + 1.

hashgt23el:
> A set with more than two elements has at least three different elements.

### CODE REVIEW

**Review by icecream17 — COMMENTED** [review]
[in the words of avekens](https://github.com/metamath/set.mm/pull/2695#pullrequestreview-1042244153), very nice, though two small notes

(approve when this is done)

File: set.mm [hunk 1]
```diff
@@ -145922,6 +145922,30 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
     AJKZLMNZOUIUJULUMUKJKZMNZUNUHUGAUKPTOZULUPOZABQUGUKDEUQURBRAUKSUAUBUJUOLUMM
     BCUCUDUEUF $.
 
+  $( The size of the union of a set with a disjoint singleton is the extended
+     real addition of the size of the set and 1, analogous to hashunsng.
```
[the remark this answers is not in the document]
Comment (BTernaryTau) [reply set.mm:145926]: Updated

File: set.mm [hunk 2]
```diff
@@ -146201,6 +146225,38 @@ are used instead of sets because their representation is shorter (and more
       HVSGXNXLSXGXTWOWLXTXFBCVQTTVTWMHWAWBWCWDWEWFWGWHWIUNWJ $.
   $}
 
+  ${
+    $d W a $.  $d V a b c $.
+    $( In a set with more than two elements are three different elements.
```
[the remark this answers is not in the document]
Comment (BTernaryTau) [reply set.mm:146230]: If we're not worried about keeping the descriptions consistent, then I'll gladly switch to yours.

**Review by icecream17 — APPROVED** [review]

**Review by tirix — APPROVED** [review]

File: set.mm (hunk 2 above)
[the remark this answers is not in the document]
Comment (jkingdon) [reply set.mm:146230]: I like the "alternative version". I'd also be in favor of changing the wording at hashgt12el2 similarly, but you can consider that optional in terms of whether it should be part of this pull request.

**Pull request merged**